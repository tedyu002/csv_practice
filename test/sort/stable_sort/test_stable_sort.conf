CSV_INPUT=test.input
CSV_OUTPUT=test.out
CSV_ERROR=test.err
CSV_RESULT=test.res
HEADERS=H1 INTEGER, H2 INTEGER
SORT_HEADERS=H1
SORT_ORDER=ASC
