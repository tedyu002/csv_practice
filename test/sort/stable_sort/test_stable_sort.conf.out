1,19
1,17
1,15
1,13
1,11
1,9
1,7
1,5
1,3
2,18
2,16
2,14
2,12
2,10
2,8
2,6
2,4
2,2
